### PR TITLE
Increase of riot shield durability

### PR DESCRIPTION
**Pull request by Elysium206**
Increases the total health of riot shields, as well as modifying their damage co-efficient to closer reflect the shield type

## About the PR
 This PR aims to make security use riot shields more often, while making the riot shield + pistol combo more viable in combat than its meager state currently. right now security never really uses them as the tradeoff of a bulky, hard to carry shield for minimal protection is rough!

The recent change to make most guns require wielding to accurately fire is already a fantastic change for gunplay. I believe increasing the usefulness of shields would also aid in making gunplay feel more varied

I also would LOVE to see security use riot shields to block hallways, section off areas, and use the anchoring system more which is why I increased the active block resist on all the riot shields

## Why / Balance
 I changed this in the hopes of not only increasing the use of riot shields, but it might also decrease the need to arm security from armory if they have access to an easier alternative on blue alert. But as stated above it also increases the variety of viable "loudouts" for combat using already existing items and abilities

I considered possibly making each shield weak to a damage type as well (I.E the standard riot shield would take 150-200% heat damage or something) to try and maintain balance. If that needs to be done it certainly could be

## Technical details
 in Shields.yml i increased the damage threshold's by 60 damage, making sure to keep the higher one that doesnt spawn materials so that weird bugs don't occur upon overkill (nukes)

I also modified the 3 riot shields damage co-efficients, making them uniform across the board for their respective damage types

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
**I believe this PR does not require media as it is entirely numerical**

**Changelog**
 

<Make sure to take this Changelog template out of the comment block in order for it to show up.>

:cl:
- tweak: Increased Security Riot shields durability significantly and makes actively blocking better

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Shields/shields.yml [hunk 1]
```diff
@@ -70,6 +70,18 @@
...
+        coefficients:
+          Blunt: 0.7
+          Slash: 0.7
+      activeBlockModifier:
+        coefficients:
+          Blunt: 0.5
+          Slash: 0.5
+        flatReductions:
+          Blunt: 1
+          Slash: 1
```
[the remark this answers is not in the document]
Comment (Elysium206) [reply Resources/Prototypes/Entities/Objects/Shields/shields.yml:84]: The nerf to the polycarbonate riot shields was intended as I would think a melee based riot shield which is see through would be poor against bullets and lasers.

As for the damage coefficient I am pretty sure it's the percentage of damage they take, similar to the damage coefficient for species.

**Comment by keronshb** [comment]
How many hits will it take for a riot shield to break while blocking when attacked by:
- standard laser
- any nukie gun of choice
- an energy sword

**Comment by Elysium206** [comment]
@keronshb 

All testing listed below was done with a human urist and a standard variety "riot shield". these numbers would change slightly depending on the type of riot shield but not super significantly

Laser rifle: Shield breaks after 15 shots, urist crits at 15 shots as well

c-20: shield breaks after 13 shots, urist also crits at 13 shots

Energy Sword: shield breaks after 8 swings, urist will crit with 7 swings

**Comment by keronshb** [comment]
How much of a difference is that to current?

Also fix the CL then this should be good to go

**Pull request merged**